Makefile

```diff
@@ -4,7 +4,11 @@
 PROJECT_NAME ?= provider-upjet-digitalocean
 PROJECT_REPO ?= github.com/crossplane-contrib/$(PROJECT_NAME)
 
-export TERRAFORM_VERSION ?= 1.8.1
+export TERRAFORM_VERSION ?= 1.5.7
+
+# Do not allow a version of terraform greater than 1.5.x, due to versions 1.6+ being
+# licensed under BSL, which is not permitted.
+TERRAFORM_VERSION_VALID := $(shell [ "$(TERRAFORM_VERSION)" = "`printf "$(TERRAFORM_VERSION)\n1.6" | sort -V | head -n1`" ] && echo 1 || echo 0)
```

Comment:
Sorry team 😭

Reply:
no worries @yordis! i'm a big believer of "[People Don’t Fail, Processes Do](https://www.lean.org/the-lean-post/articles/people-dont-fail-processes-do/)", so sorry the framework/template enabled this version in the first place 😇 

Thank you for reviewing and supporting this PR quickly! 🙇‍♂️